bucket/promtail.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": "1.3.0",
+  "description": "log agent for Loki",
+  "homepage": "https://grafana.com/loki",
+  "license": "Apache-2.0",
+  "architecture": {
+    "64bit": {
+      "url": "https://github.com/grafana/loki/releases/download/v1.3.0/promtail-windows-amd64.exe.zip",
+      "hash": "988e1c314101a5afbd4dd322e396dff3fc7e2feece1cb5c299c70321f1135ce5"
+    }
+  },
+  "bin": [
```

Comment:
```suggestion
```

Reply:
`bin` is not needed?

Reply:
https://github.com/lukesampson/scoop-extras/pull/3527#discussion_r370918023 ...